src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj

```diff
@@ -23,6 +23,9 @@
     <Compile Include="$(CommonTestPath)\System\TypeBuilderExtensions.cs" />
   </ItemGroup>
   <ItemGroup>
+      <CodeCoverageAssemblies Include="Microsoft.CSharp" /> 
```

Comment:
(I really have to learn me some msbuild. I must admit I just follow what I see elsewhere when it comes to that).